backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -185,6 +185,16 @@ public void draw (CGRect rect) {
 		lastFrameTime = System.nanoTime();
 		framesStart = lastFrameTime;
 
+		// enable OpenGL
+		makeCurrent();
+		// OpenGL glViewport() function expects backbuffer coordinates instead of logical coordinates
+		gl20.glViewport(0, 0, screenBounds.backBufferWidth, screenBounds.backBufferHeight);
+
+		String versionString = gl20.glGetString(GL20.GL_VERSION);
+		String vendorString = gl20.glGetString(GL20.GL_VENDOR);
+		String rendererString = gl20.glGetString(GL20.GL_RENDERER);
+		glVersion = new GLVersion(Application.ApplicationType.iOS, versionString, vendorString, rendererString);
```

Comment:
Shouldn't this just be called once on creation? Maybe we can make a call from `didFinishLaunching`.

Reply:
Wasn't it moved directly? In the `if (!created)` block it was `gl20.glViewport(0, 0, screenBounds.backBufferWidth, screenBounds.backBufferHeight);` too